lang/src/main/kotlin/org/partiql/lang/eval/ExprValueExtensions.kt

```diff
@@ -69,6 +69,11 @@ import java.util.TreeMap
 import java.util.TreeSet
 import kotlin.math.round
 
+const val MISSING_ANNOTATION = "\$missing"
+const val BAG_ANNOTATION = "\$bag"
+const val DATE_ANNOTATION = "\$date"
+const val TIME_ANNOTATION = "\$time"
```

Comment:
I think `ExprValue.MISSING_ANNOTATION` looks a bit weird, since it is actually annotation for `IonValue`

Reply:
Ok, I concede that neither of these locations is a good home for these constants.
Nor is `org.partiql.lang.eval` package overall, since these annotations are about interfacing with the Ion world, not about `eval`uating PartiQL.
TBD.